assets/cubyz/tools/chisel.zig.zon

```diff
@@ -0,0 +1,65 @@
+.{
+	.blockTags = .{.mineable, .choppable},
+	.disabled = .{
+		0, 0, 1, 1, 1,
+		0, 0, 0, 1, 1,
+		1, 0, 0, 1, 1,
+		1, 1, 1, 0, 1,
+		1, 1, 1, 1, 0,
+	},
+	.optional = .{
+		0, 0, 0, 0, 0,
+		0, 1, 1, 0, 0,
+		0, 1, 1, 0, 0,
+		0, 0, 0, 1, 0,
+		0, 0, 0, 0, 0,
+	},
+	.parameters = .{
+		.{
+			.source = .massDamage,
+			.destination = .damage,
+			.matrix = .{
+				1.5, 1.0, 0x0, 0x0, 0x0,
+				1.0, 0.5, 0.5, 0x0, 0x0,
+				0x0, 0.5, 0.5, 0x0, 0x0,
+				0x0, 0x0, 0x0, 0.1, 0x0,
+				0x0, 0x0, 0x0, 0x0, 0.1,
```

Comment:
I think mass damage should be distributed equally, due to how you'd swing a chisel